app/views/shopify_app/shared/redirect.html.erb

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="<%= I18n.locale %>">>
 <head>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <base target="_top">
   <title>Redirecting…</title>
+  <script src="https://unpkg.com/@shopify/app-bridge@^1"></script>
+  <script>
+    window.apiKey = "<%= ShopifyApp.configuration.api_key %>";
+    window.shopOrigin = "https://<%= current_shopify_domain %>";
```

Comment:
If we stop setting this here, we can stop removing it on the other side.

Reply:
I prefer to keep `https://` for consistency. The app gem is doing it in other pages.